.github/workflows/android-build.yml

```diff
@@ -76,6 +76,10 @@ jobs:
           path: example/android/app/build
           key: ${{ runner.os }}-build-${{ env.cache-name }}-example-${{ hashFiles('**/react-native/package.json') }}
   
+      - name: Build Android app
+        working-directory: example/android
+        run: ./gradlew assembleDebug
+  
       - name: SKDs - download required images
         run: $ANDROID_HOME/cmdline-tools/latest/bin/sdkmanager "system-images;android-30;default;x86_64"
```

Comment:
According to docs the emulator startup time can be greatly reduced by caching, see [reactivecircus/android-emulator-runner](https://github.com/ReactiveCircus/android-emulator-runner#usage--examples)

Any reason doing this manually?